xml/System.Runtime.CompilerServices/ConditionalWeakTable`2.xml

```diff
@@ -588,9 +597,20 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that can be used to iterate through the <see cref="T:System.Runtime.CompilerServices.ConditionalWeakTable`2" /> table.</summary>
+        <returns>An enumerator that can be used to iterate through the <see cref="T:System.Runtime.CompilerServices.ConditionalWeakTable`2" /> table.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This member is an explicit interface member implementation. It can be used only when the <xref:System.Runtime.CompilerServices.ConditionalWeakTable&lt;TKey,TValue&gt;> instance is cast to an <xref:System.Collections.IEnumerable> interface.
```

Comment:
```suggestion
This member is an explicit interface member implementation. It can be used only when the <xref:System.Runtime.CompilerServices.ConditionalWeakTable%602> instance is cast to an <xref:System.Collections.IEnumerable> interface.
```